jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/workitem/WorkbenchRepositoryEventListener.java

```diff
@@ -160,12 +158,14 @@ public void onServiceTaskDisabled(RepoData service) {
     }
 
     @Override
-    public void onServiceTaskInstalled(RepoData service, String target, List<String> parameters) {
-        
+    public void onServiceTaskInstalled(RepoData service, String target, List<String> parameters, String branchName) {
         try {
-            
+            // default to master
+            if(branchName == null) {
```

Comment:
missing space after if (many occurrences in the PR)

Reply:
all formatting errors are being caught during the build, this is not considered as an error, so if it is maybe we should add it as one of the formatting rules? jbpm-wb is very rigorous about formatting and it lets this pass.